### DISCUSSION TITLE
Soldier FSM and H&M

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1555] [Question and answer]
Hello is this mod compatible with Hearts&Minds https://steamcommunity.com/sharedfiles/filedetails/?id=2825929474 ?

AI are having situational awareness of a PTSD soldier, not doing patrols

Cheers.

**Comment by XDSenDX** [comment]
"It's one of the last AI mod to be still in development." That's what they said.

The mod has lots of good features but when pairing with Hearts&Minds they feel really stupid.

**Comment by Vdauphin** [comment]
> Hello is this mod compatible with Hearts&Minds https://steamcommunity.com/sharedfiles/filedetails/?id=2825929474 ?

No idea, you may want to read this: https://github.com/Vdauphin/HeartsAndMinds/discussions/1077
